src/SourceBuild/tarball/patches/fsharp/0001-Update-System.Memory-to-4.5.5.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Logan Bussell <loganbussell@microsoft.com>
+Date: Fri, 24 Jun 2022 14:57:53 -0700
+Subject: [PATCH] Update System.Memory to 4.5.5
+
+Prevents package downgrade error.
+---
```

Comment:
@lbussell - Please reference an issue or pull request that tracks backporting for all new patches.